app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -0,0 +1,276 @@
+# -*- encoding : utf-8 -*-
```

Comment:
How much of this file is just a copy paste from Blacklight and how much is actually overriding their functionality? If we could cut it down to the bare minimum it would lower our maintenance costs long-term.